From: "Francesco Poli (wintermute)" <invernomuto@paranoici.org>
Date: Mon, 30 Apr 2018 18:36:50 +0100
Subject: raster regression

fix regression on raster image rendering

This patch reverts one line from commit
430d38e06cea152e74d26b9bc5fe6aa3a5f86169
Bug-Debian: https://bugs.debian.org/888677
Last-Update: 2018-04-30
---
 src/classes/renderer/pdf.vala | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/classes/renderer/pdf.vala b/src/classes/renderer/pdf.vala
index 37690ae..d492819 100644
--- a/src/classes/renderer/pdf.vala
+++ b/src/classes/renderer/pdf.vala
@@ -120,8 +120,7 @@ namespace pdfpc {
             cr.scale(this.scaling_factor, this.scaling_factor);
             cr.translate(-metadata.get_horizontal_offset(this.area),
                 -metadata.get_vertical_offset(this.area));
-            page.render_for_printing_with_options(cr,
-                Poppler.PrintFlags.DOCUMENT);
+            page.render(cr);
 
             // If the cache is enabled store the newly rendered pixmap
             if (this.cache != null) {
